Dockerfile

```diff
@@ -0,0 +1,56 @@
+# 1: Build the exe
+FROM rust:1.56 as builder
```

Comment:
Is it possible to use rust stable as the builder? That way we don't have to update it, and new releases of rust should be backwards compatible.

Reply:
Also, what does `as builder` do?

Reply:
as builder is just a naming of the layer. And yes I will switch to stable

Reply:
Is there a benefit to naming layers? I'm a total docker noob, so don't understand what it does.

Reply:
Rust doesnt have a stable tag. I will use slim

Reply:
Slim seems to fail to build agora.